x/wasm/module.go

```diff
@@ -131,10 +133,11 @@ func (AppModule) QuerierRoute() string {
 func (am AppModule) InitGenesis(ctx sdk.Context, cdc codec.JSONMarshaler, data json.RawMessage) []abci.ValidatorUpdate {
 	var genesisState GenesisState
 	cdc.MustUnmarshalJSON(data, &genesisState)
-	if err := InitGenesis(ctx, am.keeper, genesisState); err != nil {
+	validators, err := InitGenesis(ctx, am.keeper, genesisState, am.validatorSetSource, am.Route().Handler())
+	if err != nil {
 		panic(err)
 	}
-	return []abci.ValidatorUpdate{}
+	return validators
```

Comment:
Cool, I didn't realize we could return this - thus surprised by the return value. We were always just returning nil.

Nice that you use this :+1: